recipes-core/udev-extraconf/udev-extraconf_%.bbappend

```diff
@@ -0,0 +1,16 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
this is not necessary since $thisdir/pn is a default search path.

Reply:
If I remove this line I get:

```
ERROR: udev-extraconf-1.1-r0 do_fetch: Fetcher failure for URL: 'file://0001-udev-extraconf-allow-labels-and-UUID-s-in-mount-blac.patch;striplevel=5'. Unable to fetch URL from any source.
```

Reply:
strange: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-FILESPATH

Reply:
I do not see it saying there that it is not needed when you bbappend. https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-FILESEXTRAPATHS even says to do what we do in our bbappends. And they seem to do it in poky etc.

Anyway, if you still think it should work without and can figure out why it does not (I can not, I'll remove it. :)

Reply:
no, I don't care. it was just a small tip.